# Config for the conversion topics and the message limits
conversion:
  topics:
    origin_cmd_topic: /navigation/dv_control_target
    target_cmd_topic: /fssim/cmd

    origin_odom_topic: /fssim/base_pose_ground_truth
    target_odom_topic: /slam/output/odom
    
  limits:
    max_target_speed: 5
    max_steering_angle: 90
